tests/playwright/specs/user.js

```diff
@@ -0,0 +1,25 @@
+const { CreateAccountPage } = require( '../pageobjects/index.js' );
+const { getTestString } =
+ require( '../helpers/Util.js' );
```

Comment:
remove the `.js`

Reply:
@vidhi-mody, eslint expects the import to be /Util.js. There is an eslint error when I remove the .js

Reply:
It has been fixed, added eslint lint rule.

Reply:
I have added eslint ignore rule. It should be fine now